yaml/boundaries.yaml

```diff
@@ -131,29 +131,53 @@ filters:
   #    `unrecognized_*` (where XX is the viewpoint / country code and * is the
   #    kind of boundary it is).
   #
-  - filter: {admin_level: '2', boundary: claim, claimed_by: true, geom_type: line}
+  - filter:
+      all:
+        - geom_type: line
+        - admin_level: '2'
+        - boundary: claim
+        - any:
+            - claimed_by: true
+            - disputed_by: true
     min_zoom: 8
     output:
       <<: *output_properties
       kind: unrecognized_country
```

Comment:
This might be better as `kind: disputed_claim` to match the low zoom Natural Earth zooms.

For good measure you could add `disputed: True` to the output, but it might also be worth revisiting that concept for this effort.

Reply:
To make QA and data assertions easier, suggest exporting `brk_a3: {col: ne:brk_a3}`.